packages/cxl-institute/scss/global.scss

```diff
@@ -0,0 +1,226 @@
+@import "@conversionxl/cxl-ui/scss/utils/mixins";
+@import "@conversionxl/cxl-ui/scss/cxl-theme";
+
+body,
+html {
+  overflow: hidden;
+}
+
+body {
+  background: #f6f6f6;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-areas: "nav app";
+  grid-template-columns: auto 100%;
+  height: 100%;
+}
```

Comment:
I don't get this part - layout component should define the whole grid inside, not depend on some outside global grid.

Reply:
icon-nav is outside the app layout.

Reply:
Since when? Icon nav is very much part of institute layout. Nothing else uses it, why would it be outside?

Reply:
EDIT I can already see this thinking is integrated in https://github.com/conversionxl/aybolit/pull/10